test/inspector/inspector-helper.js

```diff
@@ -544,3 +544,38 @@ exports.mainScriptSource = function() {
 exports.markMessageNoResponse = function(message) {
   message[DONT_EXPECT_RESPONSE_SYMBOL] = true;
 };
+
+exports.debugScriptAndAssertPausedMessage = function(script, assertFn) {
+  this.startNodeForInspectorTest(runTests, '--inspect-brk', script);
+
+  function runTests(harness) {
+    // Wait for the child to start running the script
+    setTimeout(() => run(), 1000);
```

Comment:
Actually, the timeout should not be needed now. The code setting up the inspector session seems to handle this wait already. I'll remove the timeout shortly.